bundles/core-commands/commands/commands.js

```diff
@@ -49,6 +49,12 @@ module.exports = (srcPath) => {
         }
       }
 
+      // Command Usage
+      Broadcast.sayAt(player);
+      Broadcast.sayAt(player, "<bold><white>                   Usage</bold></white>");
+      Broadcast.sayAt(player, "<bold><white>===============================================</bold></white>");
+      Broadcast.sayAt(player, "command <required> [optional]");
+      
```

Comment:
I think it'd be better to just put this in the `help` helpfile under where it says `help [command]` something like "Command usage will show arguments as either <required> or [optional], e.g., look [target]"